cmd/login.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/netflix/weep/pkg/util"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	loginCmd.PersistentFlags().BoolVarP(&noOpen, "no-open", "x", false, "print the link, but do not open a browser window")
+	rootCmd.AddCommand(loginCmd)
+}
+
+var loginCmd = &cobra.Command{
+	Use:   "login",
```

Comment:
What do you think of calling this `weep console` instead of `weep login`? Seems like `login` could be a bit confusing -- as a user, I might assume that it's the command to log in to ConsoleMe.

Reply:
A fun fact: I actually had called it `weep console` previously, but changed it as I thought `console` might have too much overload with `open`, since the latter aims to open resources in the console as well.

I am quite OK with switching it back to `console`. After some deliberation, I can't really think of a better term for the command.

Reply:
Yeah, this has me wondering if we should change `open` to only open things in the ConsoleMe UI and make `console` support opening resources in the AWS console as well. Any thoughts?

Regardless, I don't see a need to hold up this PR. I say we change this to `console` and have further discussion on where to go from here.